### ISSUE TITLE
Cannot select camera in greenscreen modal with tab

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2330]
**Describe the bug**

In green screen ``Choose a camera`` modal, cannot tab to the camera options; tabbing switches focus to the close button, and doesn't ever navigate to ``Green background`` or other options

**To Reproduce**
1. Go to editor
2. Click on cogwheel then Green screen option
3. try to tab, see that the only accessible option is the close button

**Expected behavior**
should be able to tab through green screen options

**Screenshots**

![2019-08-02 14 34 31](https://user-images.githubusercontent.com/5615930/62400156-16ae6c00-b533-11e9-8453-d080e9aa83fb.gif)
